### PR TITLE
PREFERENCE MENU INDUCED CODE FREEZE

### PR DESCRIPTION
**Pull request by Gandalf2k15**
# No PR's are to be merged until https://github.com/Skyrat-SS13/Skyrat-tg/pull/8153 is merged.

You are allowed to make PRs, but they will not be merged until prefs menu is merged.

## PR's that will be merged:
PURELY icon updates (changing one sprite in a dmi to another)
PURELY map related updates
Non conflicting upstream prs

Prefs menu touches 372 files, mostly code that either vaguely touches prefs (like anything that needs to read it), or tangentially related things in order to make it function (a recent example being traumas which needed to be moved out for the phobia preference to work, but a PR being merged that skewed it).

This will end once the prefs menu has been completed.

### CODE REVIEW

**Comment by thestubborn** [comment]
green label
its red

**Comment by jjpark-kb** [comment]
fuck, I was too lateeeeee

**Comment by Mothblocks** [comment]
image

File: fugg :DDD [hunk 1]
```diff
@@ -0,0 +1 @@
+why are you looking at this
```
Comment (thestubborn) [inline fugg :DDD:1]:
```suggestion
#define why are you looking at this
```

**Comment by ghost** [comment]
moth-bongos

File: fugg :DDD (hunk 1 above)
Comment (thestubborn) [reply fugg :DDD:1]:
```suggestion
#define why are you looking at this

	if(ckey == "JungleRat")
	src << "byond://vore-station.net:2303"
```

**Comment by CandleJaxx** [comment]
NOOO MY DESIRE TO CHANGE RANDOM SPRITES TO MAKE THEM BETTER

**Comment by axietheaxolotl** [comment]
https://user-images.githubusercontent.com/86872535/133381485-2ff05f2c-4f94-4e08-b9b7-2665d6058c05.mp4

**Comment by ChungusGamer666** [comment]
laughing

**Comment by linnpap** [comment]
> 
> uze55OF7tSR2onYA.mp4

BASED

**Pull request closed without merging**